Plugins/HaxeCheckStyle/CheckStyleLinter.cs

```diff
@@ -110,6 +123,28 @@ LintingResult ParseLine(string line)
                 return result;
             }
 
+            if (line.StartsWithOrdinal("Error"))
+                TraceManager.Add(line);
+            return null;
+        }
+
+        static string FindHaxelibPath()
+        {
+            string command = "$(CompilerPath)\\haxelib.exe";
+            command = PluginBase.MainForm.ProcessArgString(command);
+
+            if (File.Exists(command)) return command;
+
+            command = Path.Combine(Environment.ExpandEnvironmentVariables("%HAXEPATH%"), "haxelib.exe");
+
+            if (File.Exists(command)) return command;
+
+            foreach (var path in Environment.ExpandEnvironmentVariables("%PATH%").Split(';'))
```

Comment:
I'm not sure this is wanted.

Reply:
I added it because for example on my pc, HAXEPATH is not set either, but if you think I should remove it, I'm fine with that too.